charts/datarepo-ui/templates/configmap.yaml

```diff
@@ -27,11 +27,11 @@ data:
 
         # add a Content-Security-Policy, with some variables to make formatting nice
         set $DEFAULT "default-src 'self'";
-        set $SCRIPT "script-src 'self' 'unsafe-inline' apis.google.com *.gstatic.com *.broadinstitute.org";
+        set $SCRIPT "script-src 'self' 'unsafe-inline' apis.google.com *.gstatic.com *.broadinstitute.org *.terra.bio *.envs-terra.bio";
         set $STYLE "style-src 'self' 'unsafe-inline' fonts.googleapis.com";
-        set $IMG "img-src 'self' lh3.googleusercontent.com";
+        set $IMG "img-src 'self' *.googleusercontent.com";
```

Comment:
Looking ahead, we may want *.terra.bio *.envs-terra.bio here too in case we want to share assets.  Maybe *.gstatic.com as well?

Also, you may want to check the swagger-ui.html path to make sure the swagger UI page works ok

Reply:
It's worth checking swagger but I'm pretty sure all the content there is self hosted. The UI elements are all SVG.

Reply:
Swagger UI is all `self`. I'll add the other image sources.

Reply:
The non-prod terra environments are at `bvdp-saturn-{env}.appspot.com` (dev, alpha, staging) so we may want to include those as well.

Reply:
those are covered by `self` since TDR is running on those itself

Reply:
I don't believe TDR ever runs in appspot

Reply:
I'm not sure I understand 🤔 Those are for Terra not TDR.

Reply:
This is going to be more difficult. Since the `env` is encoded in the middle of the hostname, we can't `*` wildcard it. We shouldn't be allowing `dev` to access `alpha`, etc. Additionally, wouldn't assets be stored on something besides appspot, like a static `broadinstitute` or `google` host?

Reply:
Oh, I see. I'm not sure how that's set up, but since this is more of a "future-proof" use-case I think we can revisit this if we end up needing it.

Reply:
yeah, it might be a bit too much of a premature optimization, especially since there's no current use-case for it. Its easy enough to add if we do need to share assets in the future!